### PR TITLE
[VC-651] Support SC Android v0.0.5.5

### PR DESCRIPTION
**Pull request by shivamsharma2710**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@clevertap/clevertap-signed-call-react-native",
-  "version": "0.0.5",
+  "version": "0.0.5.1",
```
Comment (ankursaini-clevertap) [inline package.json:3]: Shouldn't we keep the version same? 0.0.5.5 here?

File: src/SignedCall.ts [hunk 2]
```diff
@@ -28,7 +33,7 @@ const eventEmitter = new NativeEventEmitter(CleverTapSignedCall);
  * @param {number} sdkVersion - The updated SDK version. /// If the current version is X.X.X then pass as X0X0X
  */
 const sdkName = 'ctscsdkversion-react-native';
-const sdkVersion = 5;
+const sdkVersion = 51;
```
Comment (ankursaini-clevertap) [inline src/SignedCall.ts:36]: Version?

Comment (shivamsharma2710) [reply src/SignedCall.ts:36]: We follow this convention for versioning:If the current version is X.X.X then we track it as X0X0X where dots get replaced with zero.

0.5.5 -> 5
0.0.5.1. -> 51
1.0.1 -> 101

In future, if we need M2P support, we can do it under 0.0.6.1_ _0.0.6.n.

Comment (shivamsharma2710) [reply src/SignedCall.ts:36]: @ankursaini-clevertap `const sdkVersion` is an integer constant used for version tracking. We push this in the appLaunched event.

For version conventions, we follow the logic below:
If the SDK version is X.X.X, we track it as X0X0X, where the dots are replaced with zeros.

For instance:
0.0.5 -> 00005 -> 5
1.0.1 -> 101

The previous SDK version was 0.0.5, so we tracked it as 5. However, for this release, we upgraded it to 0.0.5.1, hence tracking it as 51.

In the future, if we need M2P support, we can do it under versions 0.0.6.1 to 0.0.6.n, and we will track these as 61, 62, and so on.

**Review by ankursaini-clevertap — APPROVED** [review]

**Pull request merged**